test/help-create/args

```diff
@@ -1 +1 @@
-create
+create --help
```

Comment:
Bez `--help` sa testuje, ci bol najdeny projektovy adresar, ... preto som to tu upravil, aby som nemusel prerabat test.

`create` totiz vypise help, ak sa nejedna o interaktivny terminal a nevieme sa opytat, co mame vytvorit.